### DISCUSSION TITLE
Integration tool/library

### DISCUSSION THREAD

**Discussion by MessiasLima** [pacstall/pacstall#602] [Q&A] [answered]
Hello, Pacstall devs. Id like to know if there is any way to integrate with pacstall? 

Library? 
DBUS interface?
Webservice?

**Comment by Elsie19** [comment, marked as answer]
We provide a package API. https://pacstall.dev/swagger/#/repology/get_repology

**Reply by D-Brox** [reply]
We are also planning on making a library in rust, which will be used in our python rewrite

**Reply by wizard-28** [reply]
The library: https://github.com/pacstall/libpacstall/tree/oxidation

**Reply by MessiasLima** [reply]
Thank you!
